### ISSUE TITLE
Web component: Card Section- Carousel - In 200% zoom text all the elements/text present on the page are truncating from left side.

### ISSUE THREAD

**Issue by Praveen-sr** [carbon-design-system/carbon-for-ibm-dotcom#4780]
## Detailed description

Issue:
In 200% zoom text all the elements/text present on the page are truncating from left side.

Expected:
Should not truncate.

**Comment by IgnacioBecerra** [comment]
@Praveen-sr Wondering if you could take a picture of this? It looks fine on my end, but maybe I'm missing something.
It could possibly been fixed already.

cc: @RobertaJHahn

**Comment by Praveen-sr** [comment]
@IgnacioBecerra Re tested, above issue is fixed now. 

cc @RobertaJHahn

**Comment by RobertaJHahn** [comment]
Closing as this bug is fixed.
